Makefile

```diff
@@ -24,7 +24,7 @@ PKG := github.com/heptio/ark
 REGISTRY ?= gcr.io/heptio-images
 
 # Which architecture to build - see $(ALL_ARCH) for options.
-ARCH ?= linux-amd64
+ARCH ?= $(shell go env GOOS)-$(shell go env GOARCH)
```

Comment:
I realized that this will hurt `make container` if you're running on macOS. I'll see what I can do. We don't have to include the Makefile changes in this PR if it's too much of a pain.